tabbycat/results/templates/BallotsCell.vue

```diff
@@ -62,6 +66,9 @@ export default {
     needsNewBallot: function () {
       return this.viableBallotsCount === this.cellData.ballots.length
     },
+    canMergeCreate: function () {
+      return this.cellData.ballots.length > 0
+    },
```

Comment:
Both situations are intended:
1. Like at USUDC 2020, where we requested ballots from everyone even though consensus, "merging" the ballots would indicate if there is an inconsistency to be resolved.
2. Yes, as the interface allows for other ballots to be added, without being previously there. Say, tabroom enters ⅔ ballots.

Reply:
1. I see, so the idea would be that instead of clicking into one to confirm one, you'd just hit "merge" and it would either show a happy completed form, or it would refuse on the basis that they weren't all identical? Does it track by `participant_submitter` rather than by `SpeakerScoreByAdj.adjudicator`?

2. I see, makes sense!

(May want to pull in @philipbelesky to discuss how this workflow is shown to the user but that shouldn't block merging this PR.)